html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <div class="form-example">
+    <label for="email">Email: </label>
+    <div><input type="text" name="name" id="name" required></div>
```

Comment:
The `for` attribute of the label should match the `id` of the related input. Also the `name` attribute should match the data being collected, so here `"email"` would be better for the `name` attribute.